src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,3 +303,81 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () => {
+  test.beforeEach(async ({ dashboardPage, page }) => {
+    await dashboardPage.open();
+    try {
+      await checkAuthState(page);
+    } catch {
+      console.log("[E2E_LOG] - No fxa auth required, proceeding...");
+    }
+  });
+
+  test("Verify that the site footer is displayed correctly", async ({
+    dashboardPage,
+    page,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463570",
+    });
+
+    const clickOnATagCheckDomain = async (
+      aTag: Locator,
+      host: string,
+      path: string | RegExp = /.*/,
+    ) => {
+      if (typeof path === "string") path = new RegExp(".*" + path + ".*");
+      host = host.replace(/^(https?:\/\/)/, "");
+
+      const href = await aTag.getAttribute("href");
+      if (href === null) return false;
+
+      await page.goto(href);
+      const currentUrl = new URL(page.url());
+      const perceivedHost = currentUrl.hostname;
+      const perceivedPath = currentUrl.pathname;
+      expect(perceivedHost).toBe(host);
+      expect(path.test(perceivedPath)).toBeTruthy();
```

Comment:
Just for discussion sake, if you wanted the `.toHaveURL()` style-solution, it could look something like this:
```js
  // Option A: use a RegExp() object to avoid _some_ of the `/` escaping.
  const urlRe = new RegExp("support\\.mozilla\\.org/(?<locale>.*?)/kb/mozilla-monitor-faq");
  await expect(page).toHaveURL(urlRe);

  // Option B: use a /RegExp/ and escape the `/`.
  await expect(page).toHaveURL(/support\.mozilla\.org\/(?<locale>.*?)\/kb\/mozilla-monitor-faq/);
```

In both examples, that [named capturing group](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Named_capturing_group) is a bit unnecessary, but might make understanding what that unwanted path portion is. Something like this would have worked just as well:

```js
await expect(page).toHaveURL(/support\.mozilla\.org\/.*?\/kb\/mozilla-monitor-faq/);
```

Not saying we need to modify any code or anything, more of an interesting alternate concept.